spring-session/src/main/java/org/springframework/session/MapSessionRepository.java

```diff
@@ -34,6 +34,7 @@
  * </p>
  *
  * @author Rob Winch
+ * @author Eddú Meléndez
  * @since 1.0
  */
 public class MapSessionRepository implements SessionRepository<ExpiringSession> {
```

Comment:
@tsachev I think  you are right. While this would be a breaking change, it is quite minor and can be acceptable for a minor release (most users leveraging it will likely be using `SessionRepository<? extends ExpiringSession>`.

@eddumelendez Can you update this to use MapSession generic type?